apps/wallet/src/ui/app/pages/initialize/create/index.tsx

```diff
@@ -62,7 +49,7 @@ const CreatePage = () => {
 						</div>
 						<Button
 							type="submit"
-							disabled={!isValid || isSubmitting}
+							disabled={!isValid || isSubmitting || true}
```

Comment:
Is this disabled change intentional?

Reply:
yes most of the code here will be removed